customer/src/test/java/no/unit/nva/customer/events/handler/CustomerTableDynamodbStreamToEventBridgeHandlerTest.java

```diff
@@ -41,116 +48,127 @@ public class CustomerTableDynamodbStreamToEventBridgeHandlerTest {
         OTHER_STUDENT_WORK
     );
 
+    private FakeEventBridgeClient eventBridgeClient;
     private CustomerTableDynamodbStreamToEventBridgeHandler handler;
+    private ChannelClaimContext channelClaimContext;
+    private Environment environment;
+    private Context context;
 
     @BeforeEach
     void beforeEach() {
-        handler = new CustomerTableDynamodbStreamToEventBridgeHandler();
+        this.environment = mock(Environment.class);
+        when(environment.readEnv("EVENT_BUS_NAME")).thenReturn("MY_EVENT_BUS_NAME");
+
+        this.context = mock(Context.class);
```

Comment:
FakeContext

Reply:
FakeContext does not support the method I am using (getInvokedFunctionArn()). I can add support for it in new version of commons and fix that in next PR.